Concepts/Tic-Tac-Toe/Entities/Player.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicTacToe.Entities
+{
+    public class Player
```

Comment:
rowCount, colCount needs to be an array where each index represents count for a particular index. This is needed to check if current player has won in constant time.